docs/components/movement-sensor/gps/set-up-base-station.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Set up a SparkFun RTK Reference Station"
+linkTitle: "SparkFun RTK Reference Station Setup"
+weight: 20
+type: "docs"
+description: "Follow this guide to set up your own SparkFun RTK base station, a high-precision GPS module designed for use with the RTK satellite navigation technique."
+images: ["/icons/components/imu.svg"]
+# SMEs: Susmita
+---
+
+Follow this guide to set up your [SparkFun RTK Reference Station](https://www.sparkfun.com/products/22429), a high-precision GPS module designed for use with the Real-Time Kinematic (RTK) satellite navigation technique.
+RTK improves the precision of positional data from satellite-based positioning systems like the Global Positioning System (GPS).
+When placed in a known location, the RTK Reference Station calculates and transmits correction data to RTK-enabled GPS receivers based on satellite signals it receives.
+
+After following this guide, you can provide the mount point, username, password, and URL from your reference station's NTRIP server in your Viam machine config for the [`gps-nmea-rtk-pmtk`](/components/movement-sensor/gps-nmea-rtk-pmtk/) and [`gps-nmea-rtk-serial`](/components/movement-sensor/gps-nmea-rtk-serial/) models of movement sensor to allow a rover or other machine configured with your sensor to apply corrections from the reference station, enabling your machine to achieve sub-centimeter-level accuracy in positioning.
+
+## Reserve an RTK2GO casting service
+
+To distribute the correction information from your reference station, you should reserve an RTK2GO casting service for your station.
+
+Follow the instructions on [New Base Station Reservation](http://rtk2go.com/sample-page/new-reservation/) to enter information about your reference station.
+Ensure that the email you provide here is the same as the email you use to authenticate to the [Viam app](app.viam.com).
```

Comment:
@sguequierre oh sorry if it wasn't clear, I didn't mean the viam authentication email. I meant that when the user is configuring their GPS RTK, they will need to populate something called `ntrip_username` in https://docs.viam.com/components/movement-sensor/gps-nmea-rtk-serial/. That `ntrip_username` is going to be the email that they use in http://rtk2go.com/sample-page/new-reservation/. Hope that clears it up

Reply:
Ah ok thank you I was confused! Will update